tests/FSharp.Compiler.Service.Tests/MultiProjectAnalysisTests.fs

```diff
@@ -263,12 +263,12 @@ let ``Test multi project 1 xmldoc`` useTransparentCompiler =
 //------------------------------------------------------------------------------------
 
 
+type private Project = { ModuleName: string; FileName: string; Options: FSharpProjectOptions; DllName: string }
 // A project referencing many sub-projects
-module internal ManyProjectsStressTest =
+type private ManyProjectsStressTest() =
```

Comment:
Don't reuse the test project between test cases, because it touches the filesystem.